com.cassidyjames.plausible.yaml

```diff
@@ -0,0 +1,21 @@
+app-id: com.cassidyjames.plausible
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+base: io.elementary.BaseApp
+base-version: 'juno-20.08'
+sdk: org.gnome.Sdk
+command: com.cassidyjames.plausible
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --socket=session-bus
```

Comment:
This one is a bit too much, what do you need the session bus for ? You can run the app with flatpak run --log-session-bus to figure out which dbus name you need access to.

Reply:
Weird. It used to require session bus before, otherwise it wouldn't launch at all.